Project.toml

```diff
@@ -3,19 +3,20 @@ uuid = "09eb8af7-3c66-4d0b-a457-e0c10c662b2b"
 version = "0.1.0"
 
 [deps]
-IJulia = "7073ff75-c697-5162-941a-fcdaad2a7d2a"
-IOCapture = "b5f81e59-6552-4d32-b1f0-c071b021bf89"
 InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
+Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
-Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 Weave = "44d3d7a6-8a23-5bf8-98c5-b353f8df5ec9"
 
 [compat]
-IJulia = "1.23"
-IOCapture = "0.2"
-Plots = "1.19"
 Requires = "1.1"
 Weave = "0.10"
 julia = "1.5"
+
+[extras]
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+[targets]
+test = ["Test"]
```

Comment:
Why this Julia <= 1.2 way of managing test dependencies?

Reply:
Since it works and is simpler than the new, still not completely stable, test/Project.toml way (see e.g. https://github.com/JuliaDiff/ChainRules.jl/issues/574#issuecomment-1016565103).

Reply:
Hmm nice yes that's a fair critique there.